mlir/utils/performance/perfRegressionReport.py

```diff
@@ -12,10 +12,22 @@
 
 def loadMlirData(filename: str):
     df = pd.read_csv(filename, sep=',', header=0, index_col=False)
+    quickDf = pd.read_csv('quick_' + filename, sep=',', header=0, index_col=False)
```

Comment:
Would it be possible to make this optional - or to make the filename of the quick tuning report a configurable option that defaults  to `quick_ + filename`?

That makes sure that the manual regression report generation flow doesn't break

Also, this breaks on filenames such as `../rocmlir-upstream/build/perf_upstream.csv`

Reply:
... Heck, why isn't the quick tuned data being written out into the same table as the full tuned data by the perf runner?

It might be a good idea to extend the perf runner with `--quick-tuning-db=`